### ISSUE TITLE
Please provide at least a basic Readme

### ISSUE THREAD

**Issue by alexanderadam** [Stericson/busybox-free#3]
So people will get a basic overview what the project does, how the source / codebase is different from the other variants ([[1]](https://github.com/dorimanx/android-busybox-ndk) [[2]](https://github.com/PerfectButter/busybox-android) [[3]](https://www.gitorious.org/android-on-freerunner/platform_external_busybox/source/032434500668f83fe000b3ab2874b239c2a4de71:busybox/INSTALL))

Depending on what the readme contains, this issue is perhaps related to the other two issues #1 and #2. :wink:

**Comment by techdude18** [comment]
I don't really see the purpose of a readme though. The name and project structure is already self-explanatory :>

**Comment by techdude18** [comment]
If you want, send a pull. Else, this is not a issue anyways, so close this "issue"

**Comment by alexanderadam** [comment]
@techdude18 
1. to have a basic readme is *so common* that even Github will ask you to create one when you create a repository.
Actually it is even common to document any tools things in any *nix, which is why man pages are omnipresent as well. And what is "self-explanatory" may for a Java developer may not be "self-explanatory" for another person. Claiming that would probably be as arrogant as a mathematician claiming that the explicit formula for the prime count function is "self-explanatory" to someone who just knows basic math.
Even for Java developers who want to contribute it is necessary to know what rules and styleguides should be followed for a PR. There isn't even a License file in the project root.
So having a readme is pretty much a minimum standard. Larger projects usually have even more documentation than just a readme.
2. Claiming that an issue is just allowed to exist if the person who created it open an PR makes absolutely no sense. As I explained in 1.), having a Readme is pretty much a standard in open source projects. If documentation is missing, it's pretty much a bug. An open bug is one of the most legitimate reason for an open issue.

**Comment by techdude18** [comment]
Yep, we have different viewpoints i guess. Thanks for enlightening, but i don't really think that a readme is necessary. If you think that a readme is seriously needed, then make your own and send a pull then. Some developers have too many projects to handle that they dont have the time to make one

**Comment by alexanderadam** [comment]
Yes, is obvious that you have a different viewpoint. However, even if you disagree with the majority of professional developers you should at least accept that they obviously have a different opinion. Usually most developers discuss what is the minimum of information in a readme. Not whether they should _exist_. 

However, just rethink what you just wrote. Because, I would *definitely* send a PR if I would have the *expertise* for the tooling used here. Or which license Stephen was intended to use. Or which pitfalls may occur. But I don't have the expertise. I can't answer these questions.

I totally understand that not everyone has time to work on OSS but this wasn't the point here.
You just came along commenting and trolling. Nobody here demanded that a readme must be written within a few months or such.
